assets/helpers/externalLinks.js

```diff
@@ -298,6 +303,7 @@ function getWeeklyCheckout(
   cgId: CountryGroupId,
   nativeAbParticipations: Participations,
   optimizeExperiments: OptimizeExperiments,
+  promoCode?: ?WeeklyPromoCode,
```

Comment:
we've got [this `option` helper](https://github.com/guardian/support-frontend/blob/271db421a952d7a66ab4437d5d556e949e85b2a6/assets/helpers/types/option.js) that wraps the ? in a slightly more typesafe manner